### ISSUE TITLE
When listening to an episode, if I switch to another app or close the app, the audio file I was playing before should reopen

### ISSUE THREAD

**Issue by crablar** [SoftwareEngineeringDaily/software-engineering-daily-android#42]
As seen here, the app knows I have been listening

![Screenshot_20200207-065356](https://user-images.githubusercontent.com/1283090/74039503-cba86200-4976-11ea-9c04-4ecc3e79cfb2.png)

In current implementation the audio does not reopen when I close then reopen

![Screenshot_20200207-065324](https://user-images.githubusercontent.com/1283090/74039517-d662f700-4976-11ea-9394-2f77bb366bae.png)

**Comment by ashishkharcheiu** [comment]
Yes, this is a bug and should be resolved to withstand config changes for the player. The rest of the app works very fine.
